azure_media_services/templates/player.html

```diff
@@ -36,12 +36,12 @@
 
 % if download_url:
 	<li class="video-sources video-download-button">
-		<a role:"button" href="${download_url}">Download video</a>
+		<a role:"button" href="${download_url}" download>Download video</a>
```

Comment:
If I remember correctly, the download attribute isn't supported in earlier releases of IE11.  This probably won't be an issue, but wanted to call it out.